metabase/metabase
# Fix whitelabel issues with color function

Fixes https://github.com/metabase/metabase/issues/18006

How to test:
1. Login into Metabase EE
2. Check that the search bar is blue
3. Go to Admin > Whitelabel
4. Change the brand color to another one, e.g. red
5. Exit admin and go back to the home page
6. Check that the search bar is red

<img width="1049" alt="Screen Shot 2021-10-04 at 5 35 25 pm" src="https://user-images.githubusercontent.com/8542534/135870760-0ccdd94a-a4d8-46e4-a5ba-cd3ebb474319.png">
